compare_vcf.py

```diff
@@ -398,6 +493,8 @@ def summarize_results(prefix, tp, fn, fp, t, var_types, sv_length = 100, regions
                              choices = ['SNP', 'Deletion', 'Insertion', 'Inversion', 'TandemDup',
                                        'Complex', 'TransDup', 'TansDel', 'InterDup', 'Translocation'], required = False)
     main_parser.add_argument("--true_vcf", metavar="VCF", help="Input small variant sampling VCF, usually dbSNP", required = True)
+    main_parser.add_argument("--master_vcf", metavar="MASTER_VCF", help="Master whitelist, if applicable", required = False)
+    main_parser.add_argument("--call_vcf", metavar="CALL_VCF", help="Original, VCF output by variant caller, if applicable", required = False)
```

Comment:
Hi @shannog2 `--vcfs` and `--call_vcf` should be the same thing, and `--vcfs` only supports one input file for now. see https://github.com/bioinform/varsim/blob/d8d262a455e209ade7d5b401fbb3800028e129cd/compare_vcf.py#L237

Reply:
No, apologies for mixing varsim with the carrier screening pipeline, but in the pipeline when using compare_vcf for the whitelist, we have "--vcfs $wl_intersect_vcf" this does not contain the AO and RO information I need, and so this additional parameter was introduced.

We need to decide if annotating the false call vcfs could be useful for generic varsim sims. If yes, having the flexibility these additional parameters provide could be useful. If not we'll have to cancel this pull request and do something specifically for the CS pipeline.